src/dotnet/APIView/APIViewWeb/Repositories/CosmosAPIRevisionsRepository.cs

```diff
@@ -258,19 +258,19 @@ public async Task<IEnumerable<APIRevisionListItemModel>> GetAPIRevisionsAssigned
         /// </summary>
         /// <param name="crossLanguagePackageId"></param>
         /// <returns></returns>
-        public async Task<IEnumerable<string>> GetReviewIdsOfLanguageCorrespondingReviewAsync(string crossLanguagePackageId)
+        public async Task<IEnumerable<APIRevisionListItemModel>> GetLanguageCorrespondingAPIRevisionsAsync(string crossLanguagePackageId)
         {
-            var query = $"SELECT DISTINCT VALUE c.ReviewId FROM c WHERE ARRAY_LENGTH(c.Files) > 0 AND c.Files[0].CrossLanguagePackageId = '{crossLanguagePackageId}'";
+            var query = $"SELECT DISTINCT c.id, c.ReviewId FROM c WHERE ARRAY_LENGTH(c.Files) > 0 AND ARRAY_CONTAINS(c.Files, {{ \"CrossLanguagePackageId\": \"{crossLanguagePackageId}\" }}, true)";
```

Comment:
is cross language package ID unique for each revision? I assume not and it's a unique package level ID. So this query will return more than one revisions in each language. right?